packages/backend/src/util/handlers/createExpressSynthqlHandler.ts

```diff
@@ -0,0 +1,31 @@
+import { QueryEngine } from '../..';
+import { Request, Response } from 'express';
+
+type ExpressSynthqlHandler = (req: Request, res: Response) => void;
+
+export async function createExpressSynthqlHandler<T>(
+    queryEngine: QueryEngine<T>,
+): Promise<ExpressSynthqlHandler> {
+    return async (req, res) => {
+        const headers = req.headers;
+
+        const query = JSON.parse(req.body);
+
+        res.status(200);
```

Comment:
When setting `x-return-last-only` you shouldn't return 200 always as you will know in some situations when an error happens.

Reply:
Makes sense.